cdap-docs/_common/common-build.sh

```diff
@@ -291,16 +294,22 @@ function test_an_include() {
     new_md5_hash=`md5sum ${target} | awk '{print $1}'`
   fi
   
+  local result_message=""
```

Comment:
Personally, I would use an array for `result_message` so you don't constantly have to prepend additional lines with the previous contents. You simply add a new element to the array. When you set_message, you just send the entire array. That, plus printing all arguments to the functions rather than just the first should give you the same functionality, but a little cleaner.